packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/observability/probe/NewConnectionActivatedProbeTest.kt

```diff
@@ -83,7 +83,7 @@ internal class NewConnectionActivatedProbeTestForLocalEnvironment :
         isLocalhost = true,
         isEnterprise = false,
         isGenuine = true,
-        version = "7.0.14",
+        version = "7.0.12",
```

Comment:
This was unexpected, I guess they took down 7.0.14 (for whatever reason)

Reply:
Maybe. At some point we will need to update these tests to 8.0, but we don't need to do it now.